message_bus.gemspec

```diff
@@ -18,6 +18,6 @@ Gem::Specification.new do |gem|
   gem.version       = MessageBus::VERSION
   gem.required_ruby_version = ">= 1.9.3"
   gem.add_runtime_dependency 'rack', '>= 1.1.3'
-  gem.add_development_dependency 'redis'
+  gem.add_development_dependency 'redis', '3.3.5'
```

Comment:
ouch, no we can't do this, unfortunately we are cross comapatible with redis, we work with 4.0.1 as well ... revert this for now I will figure out why it is giving me grief.

Reply:
No problem. I wasn't really sure about it either. Earlier I tried updating .travis.yml to downgrade bundler to 1.15, but it didn't help.